pkg/services/serviceaccounts/api/api.go

```diff
@@ -221,6 +178,19 @@ func (api *ServiceAccountsAPI) updateServiceAccount(c *models.ReqContext) respon
 	})
 }
 
+// DELETE /api/serviceaccounts/:serviceAccountId
+func (api *ServiceAccountsAPI) DeleteServiceAccount(ctx *models.ReqContext) response.Response {
+	scopeID, err := strconv.ParseInt(web.Params(ctx.Req)[":serviceAccountId"], 10, 64)
+	if err != nil {
+		return response.Error(http.StatusBadRequest, "serviceAccountId is invalid", err)
```

Comment:
```suggestion
		return response.Error(http.StatusBadRequest, "Service Account ID is invalid", err)
```

small question of coherence with the other endpoints